src/main/java/org/xstefank/verification/EmptyFormatVerification.java

```diff
@@ -0,0 +1,13 @@
+package org.xstefank.verification;
+
+import org.xstefank.model.yaml.FormatConfig;
+
+public class EmptyFormatVerification implements Verification {
+
+    private final String MESSAGE = "Format in configuration file cannot be empty";
+
+    @Override
+    public String verify(FormatConfig formatConfig) {
+        return formatConfig.getFormat() != null ? "valid" : MESSAGE;
```

Comment:
You mean that i should directly verify the config file and not FormatConfig which loads the Mapper ? Am I right?

Reply:
yes, however I think if you are able to declare the configuration without format element than this is a valid case, it would be worth while to check if the element can be made required somehow